test/sqllogictest/temporal.slt

```diff
@@ -0,0 +1,45 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Test temporal filtering operators
+
+mode cockroach
+
+# Two columns here represent a validity interval [a, b].
+statement ok
+CREATE VIEW intervals (a, b) AS VALUES (1, 10), (1, 2), (2, 13), (3, 1)
+
+# Select out rows when each are valid.
+statement ok
+CREATE MATERIALIZED VIEW valid AS
```

Comment:
we never query `valid` - not sure if thats intentional or not it just jumped out as strange

Reply:
Huh. Weird. We should do that, because the others are just standard point queries. Will fix!